components/eventing-controller/pkg/deployment/publisher_deployment.go

```diff
@@ -0,0 +1,267 @@
+package deployment
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/kyma-project/kyma/components/eventing-controller/utils"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/kyma-project/kyma/components/eventing-controller/pkg/env"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	LivenessInitialDelaySecs = int32(5)
+	LivenessTimeoutSecs      = int32(1)
+	LivenessPeriodSecs       = int32(2)
+	BEBNamespacePrefix       = "/"
+	InstanceLabelKey         = "app.kubernetes.io/instance"
+	InstanceLabelValue       = "eventing"
+	DashboardLabelKey        = "kyma-project.io/dashboard"
+	DashboardLabelValue      = "eventing"
+	PublisherPortName        = "http"
+	PublisherPortNum         = int32(8080)
+	PublisherMetricsPortName = "http-metrics"
+	PublisherMetricsPortNum  = int32(9090)
+
+	PublisherNamespace              = "kyma-system"
+	PublisherName                   = "eventing-publisher-proxy"
+	AppLabelKey                     = "app.kubernetes.io/name"
+	PublisherSecretClientIDKey      = "client-id"
+	PublisherSecretClientSecretKey  = "client-secret"
+	PublisherSecretTokenEndpointKey = "token-endpoint"
+	PublisherSecretEMSURLKey        = "ems-publish-url"
+	PublisherSecretBEBNamespaceKey  = "beb-namespace"
```

Comment:
Do we need to export all these constants?

Reply:
Not all, I unexported the ones where it is applicable.